### PR TITLE
Feature/reg form fixes

### PR DESCRIPTION
**Pull request by rs-mgb**

### CODE REVIEW

**Comment by cod3cow** [comment]
@rolo05 Can you merge the master branch? There seem to be conflicts. Thanks.

File: src/es/components/molecules/tablePaged/TablePaged.js [hunk 1]
```diff
@@ -30,11 +30,18 @@ export default class TablePaged extends Shadow() {
       this.setParam('pageSize', this.footDropdown.value)
     }
 
-    //onklick on Search
+    //onklick on Search button
```
Comment (cod3cow) [inline src/es/components/molecules/tablePaged/TablePaged.js]:
```suggestion
    // onclick on search button
```

File: src/es/components/organisms/registerForm/RegisterForm.js [hunk 2]
```diff
@@ -253,15 +327,6 @@ export default class RegisterForm extends Shadow() {
...
       }
-      :host .form-radio-group {
-        display: flex;
-        gap: 2rem;
-      }
-      :host .form-radio-group span {
-        display: flex;
-        align-items: center;
-        gap: 0.5rem;
-      }
```
Comment (cod3cow) [inline src/es/components/organisms/registerForm/RegisterForm.js:264]: Moved these lines to web components toolbox
-> https://github.com/mits-gossau/web-components-toolbox/pull/140/files#diff-4ff7151ce55b1628a0454de3c3c339eeae89a70e0297fa38f2a9fe946ec37419R190-R198

**Review by cod3cow — CHANGES_REQUESTED** [review]
Nice, only little things to check.

File: src/es/components/molecules/tablePaged/TablePaged.js (hunk 1 above)
Comment (rs-mgb) [reply src/es/components/molecules/tablePaged/TablePaged.js]: This change isn't from me but sure I can correct it.

File: src/es/components/organisms/registerForm/RegisterForm.js (hunk 2 above)
Comment (rs-mgb) [reply src/es/components/organisms/registerForm/RegisterForm.js:264]: Lets talk about this today please, I would have some question related this. Thanks

Comment (rs-mgb) [reply src/es/components/organisms/registerForm/RegisterForm.js:264]: Sure, I have already pushed this change on Form.js but I forgot to create a PR. Thanks and thanks for PR

**Review by edmgb — APPROVED** [review]

**Pull request merged**